csrc/executor.cpp

```diff
@@ -41,6 +41,30 @@ bool shouldFillAllocationWithNan() {
   return fill_allocation_with_nan_;
 }
 
+// For scatter operator, we need initialize output tensor using self tensor.
```

Comment:
Please mention the output of a scatter op is assumed to be a fusion output as well.